Mathlib/GroupTheory/SpecificGroups/Cyclic.lean

```diff
@@ -278,7 +281,7 @@ section Classical
 
 open Classical
 
-@[to_additive IsAddCyclic.card_pow_eq_one_le]
+@[to_additive IsAddCyclic.card_nsmul_eq_zero_le]
```

Comment:
Good catch: this is a very subtle name translation!

(Same goes for the one below!)